build.sh

```diff
@@ -44,6 +44,7 @@ cmake .. \
     -DENABLE_CLANG_TIDY=OFF \
     -DENABLE_TESTS=ON \
     -DENABLE_EXAMPLES=ON \
+    -DENABLE_BENCHMARKS=ON \
```

Comment:
I don't think we should recommand others to compiler benchmarks by default.
Could you set it `OFF` here?

Reply:
this is `build.sh` /

Reply:
above DENABLE_EXAMPLES and DENABLE_TESTS all turn to ON, so I do the same

Reply:
my point is:
1. this file is not obviously recommended to build/use right?
2. the example has been set on, it's straightforward i also build the benchmark right? any special reasons to set off?
even when we build the program, those are all executable binary, we still need manual `./benchmark_xxx` to call it.

Reply:
discussed internally:
> The ENABLE_EXAMPLES && ENABLE_TESTS options are useful for development,
but ENABLE_BENCHMARKS is optional.
Having it on by default enlarges the link time for **dbms**,
so disabling it is recommended.